tools/wptserve/wptserve/utils.py

```diff
@@ -53,12 +53,12 @@ def invert_dict(dict: Dict[KT, List[VT]]) -> Dict[VT, KT]:
 
 
 class HTTPException(Exception):
-    def __init__(self, code, message=""):
+    def __init__(self, code: int, message: str = ""):
         self.code = code
         self.message = message
 
 
-def _open_socket(host, port):
+def _open_socket(host: str, port: int) -> socket.socket:
```

Comment:
Does this properly return a `socket.socket`? It looks like `socket.socket` is a function in this context (it's called on the next line)

Reply:
`socket.socket` is a class: https://docs.python.org/3/library/socket.html#socket.socket

I'm not sure if it's unusual in Python to have a lowercase class name, but in any case what we're looking at here is a constructor, so it makes sense.